src/xenia/kernel/xboxkrnl_io.cc

```diff
@@ -469,30 +469,34 @@ SHIM_CALL NtSetInformationFile_shim(PPCContext* ppc_context,
   SHIM_SET_RETURN_32(result);
 }
 
-SHIM_CALL NtQueryInformationFile_shim(PPCContext* ppc_context,
-                                      KernelState* kernel_state) {
-  uint32_t file_handle = SHIM_GET_ARG_32(0);
-  uint32_t io_status_block_ptr = SHIM_GET_ARG_32(1);
-  uint32_t file_info_ptr = SHIM_GET_ARG_32(2);
-  uint32_t length = SHIM_GET_ARG_32(3);
-  uint32_t file_info_class = SHIM_GET_ARG_32(4);
-
-  XELOGD("NtQueryInformationFile(%.8X, %.8X, %.8X, %.8X, %.8X)", file_handle,
-         io_status_block_ptr, file_info_ptr, length, file_info_class);
+struct X_IO_STATUS_BLOCK {
+  union {
+    xe::be<uint32_t> status;
+    xe::be<uint32_t> pointer;
+  };
+  xe::be<uint32_t> information;
+};
 
+dword_result_t NtQueryInformationFile(dword_t file_handle,
+                                      pointer_t<X_IO_STATUS_BLOCK>
+                                      io_status_block_ptr,
```

Comment:
hmm we should figure that out. If you install the VS plugin then the xb format script should work and you can do a hotkey in VS to format as well (makes writing code much easier). See https://github.com/benvanik/xenia/blob/master/docs/style_guide.md